android/java/org/chromium/chrome/browser/crypto_wallet/fragments/EditVisibleAssetsBottomSheetDialogFragment.java

```diff
@@ -404,10 +399,8 @@ public void onClick(View v) {
                 } catch (NumberFormatException exc) {
                 }
                 if (mSelectedNetwork.coin == CoinType.SOL) {
-                    token.isNft = isNft.isChecked();
-                    if (token.isNft) {
-                        token.decimals = 0;
-                    }
+                    token.isNft = mNftsOnly;
+                    token.decimals = 0;
```

Comment:
do we have decimals as 0 for non NFT on Solana?

Reply:
No, you're right. Let me quickly fix that.